nomos-services/data-availability/sampling/src/backend/kzgrs.rs

```diff
@@ -1,2 +1,41 @@
+use crate::backend::DaSamplingServiceBackend;
+use kzgrs_backend::common::blob::DaBlob;
+use nomos_core::da::BlobId;
+use nomos_da_network_core::SubnetworkId;
+use std::collections::BTreeSet;
 
+#[derive(Debug, Clone)]
+pub struct KzgrsSamplingBackendSettings {}
 
+pub struct KzgrsSamplingBackend {}
+
+#[async_trait::async_trait]
+impl DaSamplingServiceBackend for KzgrsSamplingBackend {
+    type Settings = KzgrsSamplingBackendSettings;
+    type BlobId = BlobId;
+    type Blob = DaBlob;
+
+    fn new(_settings: Self::Settings) -> Self {
+        todo!()
+    }
+
+    async fn get_validated_blobs(&self) -> BTreeSet<Self::BlobId> {
+        todo!()
+    }
+
+    async fn mark_in_block(&mut self, _blobs_id: &[Self::BlobId]) {
```

Comment:
In the context sampling, it seems that this could better be a `mark_complete` or similar.
 `mark_in_block` is actually happening in the consensus, this will just receive a notification about it and make actions according to it.

Reply:
Sure, this makes sense.